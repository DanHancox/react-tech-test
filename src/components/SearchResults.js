import React from "react";

function SearchResults() {
  return (
    <>
      <p>Search Results</p>
    </>
  );
}

export default SearchResults;
