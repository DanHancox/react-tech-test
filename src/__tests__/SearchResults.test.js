import { render } from "@testing-library/react";
import React from "react";
import SearchResults from "../components/SearchResults";

describe("renders search to dom", () => {
  it("renders Search", () => {
    const { asFragment } = render(<SearchResults />);

    expect(asFragment()).toMatchSnapshot();
  });
});
