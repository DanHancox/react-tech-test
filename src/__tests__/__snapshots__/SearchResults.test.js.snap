// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders search to dom renders Search 1`] = `
<DocumentFragment>
  <p>
    Search Results
  </p>
</DocumentFragment>
`;
